src/client/components/MealDetail.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState } from "react";
+import ReservationForm from "./ReservationForm";
+import MealBox from "./MealBox";
+
+function MealDetail({ match }) {
+  const [meal, setMeal] = useState({});
+
+  useEffect(() => {
+    setMeal((prev) => ({ ...prev, loading: true }));
+    fetchMeal();
+  }, []);
+
+  const fetchMeal = async () => {
+    const fetchItem = await fetch(
+      `https://meal-sharing-fp.herokuapp.com/api/meals/${match.params.id}`
```

Comment:
I can see that all your `fetch` calls have an absolute URL: `https://meal-sharing-fp.herokuapp.com`
But we would want the app to also work with your local API and database. If I were to clone down this app and run it locally it wouldn't work properly unless you provide me with your Heroku database credentials.